packages/design-toolkit/src/components/flashcard/flashcard.docs.mdx

```diff
@@ -0,0 +1,67 @@
+import { Canvas, Meta } from '@storybook/addon-docs/blocks';
+
+import * as FlashcardStories from './flashcard.stories';
+
+<Meta of={FlashcardStories} />
+
+# Flashcard
+A quick visual identifier for multiple points of data for a given map entity.
+
+## Example Usage
+```tsx
+  const details = [
+    { label: 'key', value: 'value', id: 'unique-id-1' },
+    { label: 'key', value: 'value', id: 'unique-id-2' },
+    { label: 'key', value: 'value', id: 'unique-id-3' },
+    { label: 'key', value: 'value', id: 'unique-id-4' },
+    { label: 'key', value: 'value', id: 'unique-id-5' },
+    { label: 'key', value: 'value', id: 'unique-id-6' },
+    { label: 'key', value: 'value', id: 'unique-id-7' },
+  ];
+
+  <Flashcard isLoading={isLoading}>
+    <FlashcardHero>
+      <FlashcardHeader>Identifier</FlashcardHeader>
+      <FlashcardSubheader>DATA</FlashcardSubheader>
+    </FlashcardHero>
+    <FlashcardAdditionalData>SECONDARY_DATA_01</FlashcardAdditionalData>
+    <FlashcardAdditionalData>SECONDARY_DATA_02</FlashcardAdditionalData>
+    <FlashcardDetailsList>
+      {details.map((detail) => (
+        <Fragment key={detail.id}>
+          <FlashcardDetailsLabel>{detail.label}</FlashcardDetailsLabel>
+          <FlashcardDetailsValue>{detail.value}</FlashcardDetailsValue>
```

Comment:
As exported, these are typos
```suggestion
          <FlashcardDetailLabel>{detail.label}</FlashcardDetailLabel>
          <FlashcardDetailValue>{detail.value}</FlashcardDetailValue>
```

Worth considering if you want DetailsList to be DetailList to keep it all singular or if you want to pluralize the Label and Value names